app/schemas/USER.json

```diff
@@ -0,0 +1,68 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "type": "object",
+    "properties": {
+      "userId": {
+        "type": "string"
+      },
+      "f_name": {
+        "type": "string"
+      },
+      "l_name": {
+        "type": "string"
+      },
+      "email": {
+        "type": "string"
+      },
+      "phone_number": {
+        "type": "string"
+      },
+      "location": {
```

Comment:
I wonder if we want to update this to `home_location` - that way the user can save a default home address, or use the device geo-location.